server/integration/endpoint/src/test/resources/org/infinispan/server/endpoint/endpoint-9.0.xml

```diff
@@ -18,14 +18,32 @@
         </sasl>
       </authentication>
    </hotrod-connector>
-    <hotrod-connector name="hotrod5" socket-binding="hotrod" cache-container="default">
-        <encryption security-realm="other">
-            <sni host-name="sni" security-realm="other" />
-            <sni host-name="sni2" security-realm="other2" />
-            <sni host-name="sni3" />
-        </encryption>
-    </hotrod-connector>
+   <hotrod-connector name="hotrod5" socket-binding="hotrod" cache-container="default">
+      <encryption security-realm="other">
+         <sni host-name="sni" security-realm="other" />
+         <sni host-name="sni2" security-realm="other2" />
+         <sni host-name="sni3" />
+      </encryption>
+   </hotrod-connector>
+   <hotrod-connector name="multi-tenant-1" cache-container="default" />
+   <hotrod-connector name="multi-tenant-2" cache-container="default" />
    <memcached-connector cache="memcachedCache" socket-binding="memcached" cache-container="default" idle-timeout="100" tcp-nodelay="true" worker-threads="5" receive-buffer-size="10000" send-buffer-size="10000" />
-   <rest-connector socket-binding="rest" cache-container="default" ignored-caches="cache1 cache2 cache3" context-path="/" security-domain="other" auth-method="BASIC" security-mode="READ_WRITE" extended-headers="ON_DEMAND"/>
+   <rest-connector name="rest" socket-binding="rest" cache-container="default" ignored-caches="cache1 cache2 cache3" context-path="/" security-domain="other" auth-method="BASIC" security-mode="READ_WRITE" extended-headers="ON_DEMAND"/>
    <websocket-connector socket-binding="websocket" cache-container="default" idle-timeout="100" tcp-nodelay="true" worker-threads="5" receive-buffer-size="10000" send-buffer-size="10000" />
+   <router-connector name="router" hotrod-socket-binding="router" rest-socket-binding="router" tcp-nodelay="true" keep-alive="true" receive-buffer-size="10000" send-buffer-size="10000">
```

Comment:
I have to admit I haven't looked at the xml schema yet, but looking at this example I would have expected it to be more clear.  For example why is rest and hot rod binding both named `router`?